python/lsst/ts/observatory/control/mock/mtcs_async_mock.py

```diff
@@ -229,6 +235,9 @@ async def setup_mtdome(self) -> None:
         mtdome_mocks = {
             "tel_azimuth.next.side_effect": self.mtdome_tel_azimuth_next,
             "tel_lightWindScreen.next.side_effect": self.mtdome_tel_light_wind_screen_next,
+            "cmd_park.set_start.side_effect": self.mtdome_cmd_park,
```

Comment:
this should probably be just `start` instead of `set_start`, see my comment in the code.

Reply:
bba97f5aae2eb4f18a5daf91961d9b935213a306